### DISCUSSION TITLE
is it possible to include Flow Part in a lucene index?

### DISCUSSION THREAD

**Discussion by devlife** [OrchardCMS/OrchardCore#13477] [Q&A]
I am using the Flow Part and would like to add that, if possible, to a lucene search. Is this possible?

**Comment by devlife** [comment]
After digging into this, it appears that the FlowPartIndexHandler hasn't been released yet. Do we have a timeframe for that? In the meantime, can I simply copy that code until it's released?

**Reply by jtkech** [reply]
> can I simply copy that code until it's released?

I think you can, but test it to see if it compiles and works as expected.

Otherwise 1.6 will be released soon, there is no issue and only one pending PR for the 1.6 milestone.

https://github.com/OrchardCMS/OrchardCore/pulls?q=is%3Aopen+is%3Apr+milestone%3A1.6

**Reply by devlife** [reply]
That's great! Thanks again.
